admiral/pkg/controller/common/common.go

```diff
@@ -184,3 +188,50 @@ func ConstructGtpKey(env, identity string) string {
 func ShouldIgnoreResource(metadata v12.ObjectMeta) bool {
 	return  metadata.Annotations[AdmiralIgnoreAnnotation] == "true" || metadata.Labels[AdmiralIgnoreAnnotation] == "true"
 }
+
+func GetRoutingPolicyEnv(rp *v1.RoutingPolicy) string {
+	var environment = rp.Annotations[GetEnvKey()]
```

Comment:
can we do something like this instead?
```
k := GetEnvKey()
var environment = rp.Annotations[k]
if len(environment) == 0 {
		environment = rp.Labels[k]
}
...
```

Reply:
This code is similar to what we do for other resources.